### ISSUE TITLE
Fetching events > 16 hours fails

### ISSUE THREAD

**Issue by faustus25** [MISP/PyMISP#369]
Running "last.py -l 17h" fails but works when set to 16h or less:

````
{'Connection': 'keep-alive', 'content-type': 'application/json', 'Accept': 'application/json', 'Content-Length': '39', 'User-Agent': 'PyMISP 2.4.102 - Python 3.5.2', 'Accept-Encoding': 'gzip, deflate', 'Authorization': 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'}
Request body:
{"returnFormat": "json", "last": "17h"}
Response (if any):
{'message': 'An Internal Error Has Occurred.', 'name': 'An Internal Error Has Occurred.', 'url': '/events/restSearch'}
DEBUG [api.py:271 - _check_response() ] {
    "message": "An Internal Error Has Occurred.",
    "errors": [
        "500 exception: {'message': 'An Internal Error Has Occurred.', 'name': 'An Internal Error Has Occurred.', 'url': '/events/restSearch'}"
    ],
    "name": "An Internal Error Has Occurred.",
    "url": "/events/restSearch"
}
No results for that time period
````

Have you seen similar behaviour or can something be reconfigured to retrieve events greater than 16 hours?

**Comment by Rafiot** [comment]
Are they published?

**Comment by github-pba** [comment]
Can't confirm the issue.

I created a new event in MISP, published it (!), and ran the command. The event is reported back.

> [{"Event": {"id": "1720", "orgc_id": "1", "org_id": "1", "date": "2019-07-19", "threat_level_id": "3", "info": "Demo Event - manually added", "published": true, "uuid": "5d31a2c6-4da8-4b5b-875b-473b55eb40de", "attribute_count": "0", "analysis": "0", "timestamp": "1563534022", "distribution": "3", "proposal_email_lock": false, "locked": false, "publish_timestamp": "1563534047", "sharing_group_id": "0", "disable_correlation": false, "extends_uuid": "", "event_creator_email": "admin@example.com", "Org": {"id": "1", "name": "ORGNAME", "uuid": "5cdd8e19-31a8-4f52-8d82-3dc25c69efb1"}, "Orgc": {"id": "1", "name": "ORGNAME", "uuid": "5cdd8e19-31a8-4f52-8d82-3dc25c69efb1"}, "Attribute": [], "ShadowAttribute": [], "RelatedEvent": [], "Galaxy": [], "Object": []}}]

The same output is given for any other period (19h, 20l, 1d)

**Comment by Rafiot** [comment]
yep, same here. Closing the issue.
